app/docs/0.10.x/proxy.md

```diff
@@ -598,7 +599,30 @@ More information on this topic is covered in the
 
 [Back to TOC](#table-of-contents)
 
-#### 4. Response
+#### 4. Errors & retries
+
+Whenever an error occurs during proxying Kong will use the underlying Nginx
```

Comment:
Whenever an error occurs `while` proxying`,` Kong will use the underlying Nginx retries to pass the request `onto` the next upstream.